### ISSUE TITLE
Serve reports incorrect 'On Your Network:' uri when binding to a secondary ip

### ISSUE THREAD

**Issue by joelrbrandt** [vercel/serve#469]
Bug encountered with serve version 10.0.0 running on node v10.9.0 in ubuntu 18.04

**Steps to repro**

1. Have multiple ip addresses assigned to a single physical interface on host machine (e.g., suppose eth0 has primary ip address 192.168.1.10 and secondary ip address 192.168.1.11)
2. Launch serve and specify a tcp binding that uses a secondary ip address on the physical interface (e.g., `serve -l tcp://192.168.1.11:1234 foo`)

**Actual result**

Serve correctly binds to secondary ip, and is accessible only on that ip via the host device and other devices on the network. Serve incorrectly reports this in output. For example, output may be:

```
┌──────────────────────────────────────────────────┐
│                                                  │
│   Serving!                                       │
│                                                  │
│   - Local:            http://192.168.1.11:1234   │
│   - On Your Network:  http://192.168.1.10:1234   │
│                                                  │
└──────────────────────────────────────────────────┘
```

**Expected result**

```
┌──────────────────────────────────────────────────┐
│                                                  │
│   Serving!                                       │
│                                                  │
│   - Local:            http://192.168.1.11:1234   │
│   - On Your Network:  http://192.168.1.11:1234   │
│                                                  │
└──────────────────────────────────────────────────┘
```

**Comment by lydell** [comment]
For me, it always suggests 127.0.0.1. I'm also on Ubuntu 18.04.

```
❯ serve --version
10.0.2

❯ serve

   ┌───────────────────────────────────────────────┐
   │                                               │
   │   Serving!                                    │
   │                                               │
   │   - Local:            http://localhost:5000   │
   │   - On Your Network:  http://127.0.1.1:5000   │
   │                                               │
   │   Copied local address to clipboard!          │
   │                                               │
   └───────────────────────────────────────────────┘

```

It feels like it used to suggest the correct address (192.168.x.y) but no I can't find the version.

**Comment by ouxu** [comment]
For me, the IP address does not match, I'm on macOS High Sierra 10.13.6
![image](https://user-images.githubusercontent.com/19731728/50158493-3a5bd380-030f-11e9-915a-963bde258ba1.png)

**Comment by ouxu** [comment]
For me, the IP address does not match, I'm on macOS High Sierra 10.13.6.

![image](https://user-images.githubusercontent.com/19731728/50158644-a50d0f00-030f-11e9-85fd-7113ef49243f.png)

**Comment by ouxu** [comment]
For me, the IP address does not match, I'm on macOS High Sierra 10.13.6

![image](https://user-images.githubusercontent.com/19731728/50158811-13ea6800-0310-11e9-9c2c-eaa5e1bb3301.png)

**Comment by lydell** [comment]
Fixed in v10.1.2 (by #492). Yay! :tada:

**Comment by leerob** [comment]
Closing as resolved 🙏
